core/arch/arm/plat-imx/conf.mk

```diff
@@ -1,29 +1,20 @@
 include core/arch/$(ARCH)/plat-$(PLATFORM)/platform_flags.mk
 
-CROSS_COMPILE	?= arm-linux-gnueabihf-
-COMPILER	?= gcc
-
-CFG_ARM32_core ?= y
-CFG_MMU_V7_TTB ?= y
-
-core-platform-cppflags	 = -I$(arch-dir)/include
+core-platform-cppflags  = -I$(arch-dir)/include
 core-platform-subdirs += \
 	$(addprefix $(arch-dir)/, kernel mm tee sta) $(platform-dir)
 core-platform-subdirs += $(arch-dir)/sm
 
-libutil_with_isoc := y
-CFG_GENERIC_BOOT ?= y
-CFG_IMX_UART ?= y
-CFG_MMU_V7_TTB ?= y
-CFG_NO_TA_HASH_SIGN ?= y
-CFG_PM_STUBS ?= y
-CFG_SECURE_TIME_SOURCE_CNTPCT := y
-CFG_WITH_SOFTWARE_PRNG ?= y
-CFG_WITH_STACK_CANARIES := y
+$(call force,libutil_with_isoc,y)
+$(call force,CFG_ARM32_core,y)
```

Comment:
That's correct, but I am a bit reluctant about adding such statements only for consistency checking.  I'd say we should set the variables we need but not more. In fact I'm wondering if we should keep my latest commit (aa8ae99dd40e) for this very reason.

Indeed, I'm afraid this could get out of control, especially if you consider more complex dependency schemes like crypto :fearful: In other words, currently we don't have many configuration options so we could enforce dependencies manually, but at the same time is not that useful since error troubleshooting is quite simple when you get the configuration wrong. And, if one day the configuration becomes more complex, then manual checks are not really an option and we would probably need a tool (like Kconfig).